umbral-pre-wasm/Makefile

```diff
@@ -23,6 +23,9 @@ pkg: src
 	cp package.template.json pkg/package.json
 	cp LICENSE README.md pkg/
 
+	# add patches
+	cat ./patches/api-compatibility.js.txt >> pkg/pkg-bundler/umbral_pre_wasm.js
```

Comment:
Does it have to be a `.txt`? It's a minor thing, but we're losing the syntax coloring in diffs, and the editors won't pick up the language automatically too.

Reply:
It can be anything, I just wanted to be clear that it isn't actually executable in this context, but that's not too important.